mailingteams.php

```diff
@@ -41,6 +41,21 @@ function mailingteams_civicrm_enable() {
   _mailingteams_civix_civicrm_enable();
 }
 
+/**
+ * Implements hook_civicrm_entityTypes().
+ *
+ * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_entityTypes
+ */
+function mailingteams_civicrm_entityTypes(&$entityTypes) {
```

Comment:
I'm not 100% sure about this because it should be sorted via the mixin https://github.com/australiangreens/au.com.agileware.mailingteams/blob/main/mixin/entity-types-php%401.0.0.mixin.php#L14

Reply:
Thanks for pointing this out @seamuslee001 . I checked the original repo, and the mixin directory is missing entirely despite being referenced in info.xml. Go figure.